presto-proxy/src/test/java/com/facebook/presto/proxy/TestProxyServer.java

```diff
@@ -161,7 +161,7 @@ public void testSetSession()
         }
     }
 
-    @Test(timeOut = 10_000)
+    @Test(timeOut = 15_000)
```

Comment:
Is this a related change ?

Reply:
Yes, I found this test was timing out in a few of the builds with `-T 1C`.

Reply:
Why do you think this could be?  Do you think it's because the overall load is higher with this change?  In which case, do you think we could see some CI failures for existing test cases that rely on timing/sleeping?

Reply:
Can you make this change as part of this PR?

Reply:
Updated the CI command for `test-other-modules` and reverted the change to this test